contents/product-engineers/nps-vs-csat-vs-ces.md

```diff
@@ -0,0 +1,202 @@
+| [DocuSign](https://www.docusign.com/why-docusign) | 66 |
+| [Zapier](https://www.comparably.com/brands/zapier) | 64 |
+| [Slack](https://www.comparably.com/brands/slack) | 53 |
+| [Mailchimp](https://www.comparably.com/brands/mailchimp) | 47 |
+| [NetFlix](https://www.comparably.com/brands/netflix) | 46 |
+| [Twilio](https://www.comparably.com/brands/twilio) | 34 |
+| [Dropbox](https://www.comparably.com/brands/dropbox) | 27 | 
+
+### Drawbacks of NPS
+
+NPS can be tricky because it's often contextless. It gives you an indication of your users' sentiment toward your product but not much else.
+
+To counteract this, it's best to ask follow-up questions in your NPS survey, like:
+
+- Which features do you find most useful?
+- What are you finding difficult to do?
+- Is there a feature or aspect of the product that particularly disappoints you?
+- What would it take to improve your score to a 9 or 10?
+- Would you be open to a follow-up call to discuss your feedback in more detail?
+
+### When to use NPS surveys
+
+NPS is particularly handy for understanding long-term user satisfaction – as opposed to immediate or short-term feedback.
+
+With this in mind, it's best to wait for users to properly experience your app. Then, you should run NPS surveys every month or quarter to understand how their experience changes over time. 
+
+This enables you to identify trends in how your product changes are affecting your user experience.
+
+## What is CSAT?
+
+Customer satisfaction (CSAT) score measures how satisfied your users are with a specific feature or product.
+
+Responses typically range from 1-5 or 1-10 on an "extremely dissatisfied – extremely satisfied" scale.
+
+![CSAT survey](../images/blog/nps-csat-ces/csat-survey.png)
+
+CSAT is great because it can be targeted to specific parts of your app, helping you identify where exactly users feel satisfied and where they don't. This makes it easy to find out where you can improve your UX.
+
+### How to calculate CSAT score
+
+Responses are grouped into three categories:
+
+- **Positive responses** – people who respond with 4 or 5 if your scale is from 1-5 (or 8, 9, or 10 if your scale is from 1-10).
+- **Everyone else**
+
+![CSAT scale](../images/blog/nps-csat-ces/csat-score.png)
+
+Your CSAT score is the `Total number of positive responses / Total responses = % of satisfied customers`.
+
+For example, let's say you survey `100` people. `70` people give a positive score, your CSAT would be '70 / 100 = 70%`.
+
+Naturally, a higher CSAT indicates that users are likely to continue using your product. Conversely, low CSAT scores correlate with high churn rates.
+
+### What is a good CSAT score for SaaS?
+
+A good score is typically 75-85%. Anything higher is amazing, and anything lower is poor. 
+
+According to [fullview](https://www.fullview.io/blog/csat-benchmarks-by-industry#toc-csat-benchmarks-for-software-companies), the average CSAT score for SaaS apps is 78.
```

Comment:
Do we want to link to a competitor?

Reply:
They do session replays, but their use case is focused on customer support. So Im not really sure if I would classify them as a competitor